tests/wpt/metadata/html/webappapis/scripting/processing-model-2/unhandled-promise-rejections/promise-rejection-events-onerror.html.ini

```diff
@@ -1,6 +1,5 @@
 [promise-rejection-events-onerror.html]
   type: testharness
-  expected: TIMEOUT
   [Throwing inside an unhandledrejection handler invokes the error handler.]
-    expected: TIMEOUT
+    expected: FAIL
```

Comment:
Failed with `assert_equals: expected "Uncaught Error: e2" but got "e2"`.

But the test case confused me:

https://github.com/servo/servo/blob/7eb8544759d143b6622a734ab61b14af8ce55d82/tests/wpt/web-platform-tests/html/webappapis/scripting/processing-model-2/unhandled-promise-rejections/promise-rejection-events-onerror.html#L12-L27

The spec doesn't define the error message should start with `Uncaught Error: `.
Also, if the behavior is related to [report the error](https://html.spec.whatwg.org/multipage/webappapis.html#report-the-error), maybe the error message should also be derived from the thrown error? For this case, it would be just `e2`?